### PR TITLE
run molecule in parallel

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
if you wanna know the gory details, read https://www.die-welt.net/2024/04/running-ansible-molecule-tests-in-parallel/
otherwise: this speeds up the test from 26+ minutes to 13 minutes. enjoy!

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -146,10 +146,15 @@ jobs:
       - name: Install Ansible 2.9 compatible molecule and Jinja
         run: pip install --upgrade 'molecule[docker]<4.0' 'Jinja2<3.1'
         if: matrix.ansible == 'v2.9.17'
+      - name: Install collection dependencies
+        run: cd roles/backup && molecule dependency -s debian
```
Comment (ekohl) [inline .github/workflows/main.yml]: Looking at https://ansible.readthedocs.io/projects/molecule/usage/ I wonder if `-s` should be written as `--scenario-name`.

**Review by ekohl — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (evgeni) [reply .github/workflows/main.yml]: How dare you taking my "I DEMAND LONGOPTS EVERYWHERE" and point it at me?! :)

Comment (ekohl) [reply .github/workflows/main.yml]: It's Monday, it shouldn't be easy today.

**Pull request merged**